src/cli/cli.cpp

```diff
@@ -2984,7 +2984,7 @@ otError Interpreter::ProcessNetworkTime(Arg aArgs[])
 
         networkTimeStatus = otNetworkTimeGet(mInstance, &time);
 
-        OutputFormat("Network Time:     %luus", time);
+	OutputFormat("Network Time:     %lldus", time);
```

Comment:
I think this should be reverted in the official patch. The code was correct, so it must be an issue in our nano std lib if it doesn't print correctly.